settings.gradle

```diff
@@ -8,4 +8,5 @@ include 'spectator-api',
         'spectator-ext-sandbox',
         'spectator-ext-spark',
         'spectator-reg-metrics3',
-        'spectator-reg-servo'
+        'spectator-reg-servo',
+        'spectator-web'
```

Comment:
Since the is specific to spring, maybe `spectator-web-spring` would make more sense in case there winds up being `spectator-web-servlet` etc in the future.